### PR TITLE
Feature/csh 3954 slides property

### PR DESCRIPTION
**Pull request by wwsno**
This changes the format in the breaking way on the development environment!

### CODE REVIEW

File: components/components-definition.json [hunk 1]
```diff
@@ -72,6 +72,8 @@
             "icon": "icons/components/slideshow.svg",
             "label": "{{ 'DESIGN_COMPONENT_SLIDESHOW_LABEL' | translate }}",
             "properties": [
+                { "name": "slides", "directiveKey": "slideshow" },
+                { "control": { "type": "header" }, "label": "Slideshow properties" },
```
Comment (wwjhu) [inline components/components-definition.json]: Shouldn't this be a string key instead?

Comment (wwsno) [reply components/components-definition.json]: String key?

Comment (wwjhu) [reply components/components-definition.json]: Like `"label": "{{ 'DESIGN_COMPOMNENT_SLIDESHOW_PROPERTIES' | translate }}"`

Comment (wwsno) [reply components/components-definition.json]: Added as bullet to CSH-3722. There is no TMS string yet.

**Pull request merged**